code/modules/vehicle/sportscar.dm

```diff
@@ -0,0 +1,47 @@
+/obj/vehicle/car
+	name = "sports car"
+	desc = "A very luxurious vehicle."
+	icon = 'icons/vehicles/sportscar.dmi'
+	icon_state = "sportscar"
+	generic_pixel_x = 0
+	generic_pixel_y = 4
+	vehicle_move_delay = 1
+	var/static/image/carcover = null
```

Comment:
http://www.byond.com/forum/?post=137782#comment936991
Assuming the post can be trusted (which I don't doubt) in this case it means that all instances of this (sub)type will share the same carcover image.

The good part is that you'll only ever have one instance of this image, no matter how many cars there are.
The bad part is, should the image change it'll affect all cars (but this can be worked around by having a second non-static var for custom image situations).